app/service/file_svc.py

```diff
@@ -156,6 +156,19 @@ def get_payload_name_from_uuid(self, payload):
     def get_payload_packer(self, packer):
         return self.packers[packer].Packer(self)
 
+    def list_exfilled_files(self, startdir=None):
+        if not startdir:
+            startdir = self.get_config('exfil_dir')
+
+        exfil_files = dict()
+        exfil_folders = [f.path for f in os.scandir(startdir) if f.is_dir()]
```

Comment:
I went to the Exfills UI before running any operations, selected "all" and got an Internal Server Error on this line:
`FileNotFoundError: [Errno 2] No such file or directory: '/tmp/caldera'`

Reply:
yup, looks like I forgot to check that, a simple validation should help prevent that.  I'll push up a change.